contrib/packs/actions/templates/pack.yaml.j2

```diff
@@ -0,0 +1,13 @@
+---
+name: {{ name }}
+description: {{ description }}
+{% if keywords %}
+keywords:
+{% for keyword in keywords %}
```

Comment:
Do we need to use `-` at the end (-%}) to make sure there is no extra blank line where the control statement is?